### ISSUE TITLE
cake acl grant throws Unknown method "node" in ORM/Table

### ISSUE THREAD

**Issue by sarrala** [cakephp/acl#47]
All I know is that it does not load AclBehavior which seems to contain required function.

`$ bin/cake acl grant test controllers`

Here's some frames:
```
Exception: Unknown method "node" in [vendor/cakephp/cakephp/src/ORM/Table.php, line 1719]
2015-02-04 08:39:59 Error: [BadMethodCallException] Unknown method "node"

#Stack Trace:
#
# Core Framework
#
#0 vendor/cakephp/acl/src/Model/Table/PermissionsTable.php(215)			: Cake\ORM\Table->__call('node', Array)
#1 vendor/cakephp/acl/src/Model/Table/PermissionsTable.php(215)			: Cake\ORM\Table->node(Array)
#
# Acl Manager
#
#2 vendor/cakephp/acl/src/Model/Table/PermissionsTable.php(163)			: Acl\Model\Table\PermissionsTable->getAclLink(Array, 1)
#3 vendor/cakephp/acl/src/Adapter/DbAcl.php(95)							: Acl\Model\Table\PermissionsTable->allow(Array, 1, '*', 1)
#4 vendor/cakephp/acl/src/Controller/Component/AclComponent.php(132)	: Acl\Adapter\DbAcl->allow(Array, 1, '*')
#5 vendor/cakephp/acl/src/Shell/AclShell.php(270)						: Acl\Controller\Component\AclComponent->allow(Array, 1, '*')
#6 [internal function]													: Acl\Shell\AclShell->grant('test', 'controllers')
#
# Shell Dispatcher:
#
#7 vendor/cakephp/cakephp/src/Console/Shell.php(376)					: call_user_func_array(Array, Array)
#8 vendor/cakephp/cakephp/src/Console/ShellDispatcher.php(200)			: Cake\Console\Shell->runCommand(Array, true)
#9 vendor/cakephp/cakephp/src/Console/ShellDispatcher.php(171)			: Cake\Console\ShellDispatcher->_dispatch()
#10 vendor/cakephp/cakephp/src/Console/ShellDispatcher.php(122)			: Cake\Console\ShellDispatcher->dispatch()
#11 bin/cake.php(20)													: Cake\Console\ShellDispatcher::run(Array)
#12 {main}
```

**Comment by charlesschaefer** [comment]
The same here.

**Comment by charlesschaefer** [comment]
I've searched and discovered the problem.

Shell\AclShell.php calls ```\Cake\ORM\TableRegistry::get('Aros')->schema();``` without pass the className configuration in order to check if the ACL tables are created.

It should remove the default Cake ORM table created from the table registry after that.

**Comment by markstory** [comment]
Fixed in #62
